mappings/net/minecraft/block/CarvedPumpkinBlock.mapping

```diff
@@ -2,10 +2,12 @@ CLASS net/minecraft/class_2276 net/minecraft/block/CarvedPumpkinBlock
 	FIELD field_10748 FACING Lnet/minecraft/class_2753;
 	FIELD field_10749 snowGolemDispenserPattern Lnet/minecraft/class_2700;
 	FIELD field_10750 snowGolemPattern Lnet/minecraft/class_2700;
-	FIELD field_10751 IS_PUMPKIN_PREDICATE Ljava/util/function/Predicate;
+	FIELD field_10751 IS_GOLEM_HEAD_PREDICATE Ljava/util/function/Predicate;
```

Comment:
```suggestion
	FIELD field_10751 GOLEM_HEAD_PREDICATE Ljava/util/function/Predicate;
```
Don't think the `IS` prefix is necessary.

Reply:
We might need to open up an issue for consistency: [`IS_AIR_PREDICATE`](https://github.com/FabricMC/yarn/blob/78fcff22ef6dd852912008bd25a51ee5a39437bc/mappings/net/minecraft/server/command/CloneCommand.mapping#L2), [`IS_FOOD`](https://github.com/FabricMC/yarn/blob/78fcff22ef6dd852912008bd25a51ee5a39437bc/mappings/net/minecraft/entity/passive/PandaEntity.mapping#L6), [`DOOR_BREAK_DIFFICULTY_CHECKER`](https://github.com/FabricMC/yarn/blob/78fcff22ef6dd852912008bd25a51ee5a39437bc/mappings/net/minecraft/entity/mob/ZombieEntity.mapping#L2), [`BLOW_UP_FILTER`](https://github.com/FabricMC/yarn/blob/78fcff22ef6dd852912008bd25a51ee5a39437bc/mappings/net/minecraft/entity/passive/PufferfishEntity.mapping#L4), [`PREDICATE`](https://github.com/FabricMC/yarn/blob/78fcff22ef6dd852912008bd25a51ee5a39437bc/mappings/net/minecraft/entity/decoration/AbstractDecorationEntity.mapping#L3), and [`RIDERS`](https://github.com/FabricMC/yarn/blob/78fcff22ef6dd852912008bd25a51ee5a39437bc/mappings/net/minecraft/item/BoatItem.mapping#L2).

Reply:
Hope you can create the issue since you have access to the content of your comment while i don't. though personally i'd favor CHECKER/FILTER suffixes

Reply:
Ok, see #1624.